File: TD/TD2/Code/4.cpp
/* Calculer la moyenne de N valeurs saisie par USER */
#include <iostream>

int main()
{
  int a;
  int b;
  int c;
 
  c = 0;

  std::cout << "Donne moi le nombre de valeurs : " << std::flush;
  std::cin >> a;

  for (int i = 0; i < a; ++i) {
    std::cout << "Donne moi une valeur : " << std::flush;
    std::cin >> b;
    c += b;
  }

  std::cout << "La moyenne est : " << c / b << std::endl;

  return 0;
}

File: TD/TD2/Code/Python/4.py
#!/usr/bin/python
# Calculer la moyenne de N valeurs saisie par USER
somme = 0

print('Donne moi une valeur : ', end='')
a = int(input())

print('Donne moi des valeurs une par une : ', end='')
for i in range(a):
    x = int(input())
    somme += x

print('La moyenne est égal à', somme / a)
